libs/ui-watt/src/lib/components/table/watt-table.component.html

```diff
@@ -80,10 +80,10 @@
     mat-row
     *matRowDef="let row; columns: _getColumns()"
     [attr.aria-selected]="row === activeRow"
-    (click)="rowClick.emit(row)"
+    (click)="_onRowClick(row)"
```

Comment:
nitpick: probably something we can't do much about right now, but even though the click doesn't have a visual effect, change detection will still be triggered every time there's a click on a row. That's most likely not an issue when the visual cues, like mouse pointer or row highlight, are not there, but still something to be aware of.

The best, of course, is if it was possible to not set the click in the first place, but I'm not sure there's an easy solution for that. :)

Reply:
What do you mean by adding it manually?

Reply:
I feel like this is an optimization we can look into **if** it becomes relevant. But it is a good observation!

Reply:
Yes, for sure. Nothing